### ISSUE TITLE
pdb support?

### ISSUE THREAD

**Issue by mitchcapper** [BepInEx/Il2CppInterop#105]
I noticed the code used to seemingly have PDB support in it.   I can't find any reference to it anymore however.   I am trying to track down a crash with only some basic patching but don't see a good way to manually resolve the callstack call offsets using dnspy.   Should I just regenerate the old using the old unhollow or am I missing something around it?

**Comment by ds5678** [comment]
Are you referring to this?

https://github.com/knah/Il2CppAssemblyUnhollower/tree/master/UnhollowerPdbGen

**Comment by mitchcapper** [comment]
Yes, for the record I resolved my crash another way but yeah the PDB gen support seemed useful in some scenarios.

**Comment by ds5678** [comment]
https://github.com/SamboyCoding/Cpp2IL/pull/382
